content/post/the-future-is-rusty.en.md

```diff
@@ -0,0 +1,69 @@
++++
+title = 'The Future is Rust(y)?'
+slug = 'post'
+image = 'https://cdn.dribbble.com/users/241205/screenshots/4977779/drib_00_webasm_tank.png'
+date = "2019-06-01T00:00:00"
+author = 'Swarnim Arun'
+tags=['rust','prgramming']
+categories =['code-philosophy']
+description = 'My quick take on how I see Rust as the new leader of this ever changing programming world'
++++
+
+<b>Rust is the system programming language for the future.</b> Quite a presumptuous statement, said my confreres. I would be inclined to agree with them but for the experience of the agony of using C.

```

Comment:
```suggestion
**Rust is the system programming language for the future.** Quite a presumptuous statement, said my confreres. I would be inclined to agree with them but for the experience of the agony of using C.
```
Markdown needs markdown markup.

Reply:
markdown actually also supports html formatting tags and they can be helpful if you need to use ** otherwise.

But yeah here I should change them. 👍